Makefile

```
@@ -40,12 +40,12 @@ lint: ci-lint
 build: build-agent
 
 build-agent: $(BIN) src/agent #lint
-	$(GO_BUILD_VARS) go build -tags $(GO_BUILD_TAGS) -o $(BIN)/agent src/agent/main/main.go
+	GOFLAGS=-mod=mod $(GO_BUILD_VARS) go build -tags $(GO_BUILD_TAGS) -o $(BIN)/agent src/agent/main/main.go
 
 build-release: build-agent-release
```

checkmake (0.3.2)

[warning] Target "build-release" should be declared PHONY.

(phonydeclared)